docs/tutorial/part-two/index.md

```diff
@@ -435,7 +435,7 @@ directory. But, if it's used only in one file, create it inline.
 
 Modify `about-css-modules.js` so it looks like the following:
 
-```jsx{7-19,25-34}:title=src/pages/about-css-modules.js
+```jsx{7-15,21-30}:title=src/pages/about-css-modules.js
```

Comment:
Should the second range be up to `31` to include closing bracket `/>` for second `User` component?

Reply:
When I'm counting it out, I'm seeing the `/>` on 30 (I added comments for every 5th line below):

```jsx{7-15,21-30}:title=src/pages/about-css-modules.js
import React from "react"
import styles from "./about-css-modules.module.css"
import Container from "../components/container"

/*5*/ console.log(styles) 

const User = props => (
  <div className={styles.user}>
    <img src={props.avatar} className={styles.avatar} alt="" />
/*10*/    <div className={styles.description}> 
      <h2 className={styles.username}>{props.username}</h2>
      <p className={styles.excerpt}>{props.excerpt}</p>
    </div>
  </div>
/*15*/)

export default () => (
  <Container>
    <h1>About CSS Modules</h1>
/*20*/    <p>CSS Modules are cool</p>
    <User
      username="Jane Doe"
      avatar="https://s3.amazonaws.com/uifaces/faces/twitter/adellecharles/128.jpg"
      excerpt="I'm Jane Doe. Lorem ipsum dolor sit amet, consectetur adipisicing elit."
/*25*/    />  
    <User
      username="Bob Smith"
      avatar="https://s3.amazonaws.com/uifaces/faces/twitter/vladarbatov/128.jpg"
      excerpt="I'm Bob smith, a vertically aligned type of guy. Lorem ipsum dolor sit amet, consectetur adipisicing elit."
/*30*/    />  
  </Container>
)
```

Reply:
Ah, you are right - was looking at https://www.gatsbyjs.org/tutorial/part-two/ and there seems to be extra line there for some reason